### ISSUE TITLE
Add Flatpak support

### ISSUE THREAD

**Issue by Plarpoon** [WowUp/WowUp#1431]
After a long debate and discussion on a similar issue I decided to open up this one, since the solution generally agreed seems to be that we should add flatpak but without removing entirely appimage support since some people actually do enjoy it.

Flatpak is the most widely used method of distributing software on Linux other than the distro's own repositories, it's efficient and especially with something based on chromium it allows for little amount of extra bloat (since it's very likely any user already has at least another chromium based app installed) and it offers an integrated system to fully update your package automatically, feature that outside of Windows I don't think exists at the moment and would be beneficial to add.

              I presume I should add open a new issue since I think we are all generally agreeing the solution should be adding flatpak and not substituting appimage with flatpak. Or at the very least nobody seems to have anything against that.

_Originally posted by @Plarpoon in https://github.com/WowUp/WowUp/issues/1424#issuecomment-1521073791_

**Comment by Plarpoon** [comment]
I noticed that too, I didn't get any information about when it will be available nor here nor the Discord server (it's quite active, so I suggest visiting it if you want to have quicker access to information).

Anyway, the work is there, I presume it's just a matter of enabling it whenever time will be available for testing, but there might be some issues I am not aware of yet.

**Comment by DaZRool** [comment]
Any news on this matter? A official flatpak on flathub would be great!

**Comment by burgurpun** [comment]
#1546

**Comment by p3lim** [comment]
I grew tired of waiting so I created an unofficial repository with Flatpak builds: <https://github.com/p3lim/wowup-flatpak>

Once an official one is in place I'll remove it.

**Comment by matkeith** [comment]
@p3lim thanks! I was wondering how to build a flatpak version that was supposedly committed but I had no idea how to do so. I vastly prefer this as I only used appimage for this one app and would just rather not.

**Comment by aiqc** [comment]
Just booting up on Linux. Downloaded the AppImage. Thoughts: bah, where do I want to move it/ where am I supposed to keep these? I wish there was a flatpak. 

Because Faugus Launcher offers a flatppak, I think distributing wowup as a flatpak makese sense now
